docs/cli.md

```diff
@@ -34,39 +34,51 @@ in which it is being run. Currently model analyzer supports 2 modes.
-`--latency-budget`, then the best model configuration will be the one with the highest throughput in the given budget. 
+the optimal model configuration for an online inference scenario. By default in
+online mode, the best model configuration will be the one that maximizes
+throughput. If a latency budget is specified to the [profile subcommand](#subcommand-profile) via
+`--latency-budget`, then the best model configuration will be the one with the highest throughput in the given budget.
 
-In online mode the analyze and report subcommands will generate summaries specific to online inference. 
+In online mode the analyze and report subcommands will generate summaries specific to online inference.
 See the example [online summary](../examples/online_summary.pdf) and [online detailed report](../examples/online_summary.pdf).
 
 ### Offline Mode
 
 The offline mode `--mode=offline` tells Model Analyzer to operate to find the
-optimal model configuration for an offline inference scenario.  By default
+optimal model configuration for an offline inference scenario. By default
 in offline mode, the best model configuration will be the one that maximizes throughput.
-A minimum throughput can be specified to the [analyze subcommand](#subcommand-analyze)
-via `--min-throughput` to ignore any configuration that does not exceed a minimum number of inferences per second. 
+A minimum throughput can be specified to the [profile subcommand](#subcommand-profile)
+via `--min-throughput` to ignore any configuration that does not exceed a minimum number of inferences per second.
 
 In offline mode the analyze and report subcommands will generate reports specific to offline inference.
 See the example [offline summary](../examples/offline_summary.pdf) and
 [offline detailed report](../examples/offline_detailed_report.pdf) examples.
 
 ## Model Analyzer Subcommands
 
-The Model Analyzer's functionality is split across 3 separate subcommands. Each
+The Model Analyzer's functionality is split across three separate subcommands. Each
 subcommand has its own CLI and config options. Some options are required for
 more than one subcommand (e.g. `--export-path`). See the [Configuring Model
 Analyzer](./config.md) section for more details on configuring each of these
 subcommands.
 
 ## Subcommand: `profile`
 
-The `profile` subcommand allows the user to run model inferences using perf
-analyzer, and collect metrics like throughput, latency and memory usage. Use the
-following command to see the usage and argument descriptions for the subcommand.
+The `profile` subcommand begins by loading the "latest" checkpoint (if available) in
+the checkpoint directory. It will then run model inferences using perf
+analyzer, and collect metrics like throughput, latency and memory usage for
+any measurements not present in the checkpoint.
+
+Next, it sorts the models specified in the CLI or
+config YAML, using the objectives specified in the config YAML. Finally, it constructs summary PDFs
```

Comment:
the objectives -> any objectives 
(or else it implies that it is required)

Reply:
Done